### ISSUE TITLE
[Bug] multiple problems restoring

### ISSUE THREAD

**Issue by 2011** [NeoApplications/Neo-Backup#786]
**Description**

Backed of everything because I had to do a clean flash. After reinstalling Neo-Backup, I attempted to restore all user apps.

It appears that Neo-Backup never "finished." In notifications, I still see (several hours later): "Restore @ Wed 15:37:55" and beneath that "0 / 5 [icon of a single person] 0 [icon of two people] 5" I see 10 user apps on the homepage that have a strange orange android icon (and don't appear among the apps on the device. This notification keeps appearing and disappearing with no progress ever made.

When I select "View the log", I get  "The filtered list is empty."

**System Information(please complete the following information):**

- Device: Mi A3
- Android Version: 13
- ROM: (AOSP, MIUI, OOS...) : Lineage OS 20
- App's Version: 8.3.2

**Comment by machiav3lli** [comment]
What are those apps and how big are their backups?

**Comment by 2011** [comment]
As noted above, that happened on Android 13 (an attempted upgrade to a recent LineageOS version). I hated the ROM so much that I simply abandoned it, and found a suitable Android 11 ROM. Neo-Backup had no issues (beyond the usual problems such as omitting keys in the Keystore) backing up any of the apps on that ROM. Although I might run into this problem again in the future, I will close this issue for now.
